api/npls.go

```diff
@@ -0,0 +1,127 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"net/http"
+
+	"github.com/banzaicloud/nodepool-labels-operator/pkg/npls"
+	"github.com/banzaicloud/pipeline/cluster"
+	pipConfig "github.com/banzaicloud/pipeline/config"
+	"github.com/banzaicloud/pipeline/internal/platform/gin/utils"
+	pkgCluster "github.com/banzaicloud/pipeline/pkg/cluster"
+	pkgCommon "github.com/banzaicloud/pipeline/pkg/common"
+	"github.com/banzaicloud/pipeline/pkg/k8sclient"
+	"github.com/gin-gonic/gin"
+	"github.com/spf13/viper"
+)
+
+// returns NodePoolStatus map with updated user labels from NodePoolLabelSets
```

Comment:
Shouldn't GoDoc start with the name of the function? With `getNodePoolsWithUpdatedLabels` in this case?